docs: KB about using client IP to solve a specific problem (#204)
telerik/fiddler-everywhere-docs: knowledge-base/how-to-track-different-devices.md

---

> title: Capture traffic from multiple devices

Reviewer: Use title case.

description: "Learn how to capture traffic from multiple devices and how to differentiate the traffic based on the device IP."
type: how-to
slug: fiddler-multiple-devices
publish: true
res_type: kb
---

## Environment

|   |   |
|---|---|
| Fiddler Everywhere | 1.0.0 and above |
| Operating System | Windows, macOS, Linux |

## Description

> Fiddler Everywhere can simultaneously capture traffic from multiple remote computers and devices (like Android and iOS smartphones).  What configuration is needed for this to happen, and how can we differentiate the traffic from each different device?

Reviewer: ```suggestion
Fiddler Everywhere can simultaneously capture traffic from multiple remote computers and devices (like Android and iOS smartphones).  What is the needed configuration, and how can I differentiate the traffic from each individual device?
```

> ## Solution (for remote computers)

Reviewer: Use title case: 
Solution (for Remote Computers)

> **Prerequisites**

Reviewer: In general, don't use bold as a sub-section title, you can use the next heading level, for example.
In this case, I would remove the Prerequisites title because using it will require a second title for the section with the numbered list. See the suggestion below.
```suggestion
```

> - The Fiddler host and the remote computer should both be on the same network, and they should "see" each other (no firewall restrictions).

Reviewer: Don't use "should".
See https://testdocs.telerik.com/style-guide/basic-rules-and-guidelines#use-properly-should-could-must-have-to-need-to.

Reviewer: ```suggestion
Before you start, verify that the host and the remote computer are on the same network, and they "see" each other (no firewall restrictions).
```

> To capture traffic from a remote computer, follow this steps:

Reviewer: ```suggestion
To capture traffic from a remote computer:
```

1. Start Fiddler Everywhere on the host computer and enable **Live Traffic** (system capture). 

1. Ensure your are [ready to capture system traffic]({%slug capture-traffic-get-started%}#system-capturing).

1. Go to **Settings**  > **Connections** and ensure **Allow remote computers to connect** is checked.

1. On the remote computer, set the proxy settings to the machine name of the Fiddler host at port 8866.

1. In Fiddler Everywhere, filter the traffic through the **Client IP** column while using the local address of the connected remote computer.

    ![Filtering by Client IP](../images/kb/client-ip/filter-by-client-ip.png)

> ## Solution (for mobile devices)

Reviewer: Use title case.

> **Prerequisites**

Reviewer: I suggest using the same approach described in the previous section.
